### PR TITLE
Clarify expiration of certificates signed by kube-controller-manager

### PR DESCRIPTION
**Pull request by jpetazzo**
The current wording of the documentation suggests that the duration/expiration
of the certificates can be changed by asking a specific time in the CSR. While
it's technically possible to specify a duration (e.g. as a custom annotation),
there is no agreed-upon method to do so, and the built-in signer uses a fixed
expiration time anyway.

This clarifies the situation.

See kubernetes/kubernetes#92678 for discussion.

### CODE REVIEW

File: content/en/docs/reference/access-authn-authz/certificate-signing-requests.md [hunk 1]
```diff
@@ -92,8 +102,8 @@ Kubernetes provides built-in signers that each have a well-known `signerName`:
        but it is often not the only cluster-admin subject in a cluster.
     1. Permitted x509 extensions - honors subjectAltName and key usage extensions and discards other extensions.
     1. Permitted key usages - must include `["client auth"]`. Must not include key usages beyond `["digital signature", "key encipherment", "client auth"]`.
-    1. Expiration/certificate lifetime - minimum of CSR signer or request.
-       The signer is responsible for checking that the certificate lifetime is valid and permissible.
+    1. Expiration/certificate lifetime - set by the cluster signing duration configuration
+       option of the kube-controller-manager.
```
Comment (liggitt) [inline content/en/docs/reference/access-authn-authz/certificate-signing-requests.md:106]: the implementation in kube-controller-manager is one implementation of this signer, but it can be replaced by a custom implementation. I would rephrase this as "set by the `--cluster-signing-duration` option for the kube-controller-manager implementation of this signer" (same comment applies to other places)

**Comment by kbhawkey** [comment]
Page preview:
https://deploy-preview-24653--kubernetes-io-master-staging.netlify.app/docs/reference/access-authn-authz/certificate-signing-requests/

File: content/en/docs/reference/access-authn-authz/certificate-signing-requests.md (hunk 1 above)
Comment (jpetazzo) [reply content/en/docs/reference/access-authn-authz/certificate-signing-requests.md]: Indeed! I updated the language with your suggestion.

File: content/en/docs/reference/access-authn-authz/certificate-signing-requests.md [hunk 2]
```diff
@@ -78,6 +78,16 @@ that case, the documentation for the signer should specify the meaning of
 additional certificates; for example, this might be the certificate plus
 intermediates to be presented during TLS handshakes.
 
+The PKCS#10 signing request format doesn't allow to specify a certificate
+expiration or lifetime. The expiration or lifetime therefore has to be set
```
Comment (kbhawkey) [inline content/en/docs/reference/access-authn-authz/certificate-signing-requests.md:82]: nit: The PKCS#10 signing request format doesn't allow you to specify a certificate expiration or lifetime.
nit: Replace "e.g." with ",for example,"
